### PR TITLE
Refine example code in en doc

### PR DESCRIPTION
**Pull request by FrostML**
Refine the doc of create_global_var, pad, pad_constant_like and pad2d to make it consistent with cn doc. 
Related to [FluidDoc #1964](https://github.com/PaddlePaddle/FluidDoc/pull/1964)
Related to [FluidDoc #1957](https://github.com/PaddlePaddle/FluidDoc/pull/1957)

### CODE REVIEW

**Comment by FrostML** [comment]
* create_global_var:
cn_doc:
image
en_doc:
image

**Comment by FrostML** [comment]
* pad
cn_doc:
image
image

en_doc:
image
image

**Comment by FrostML** [comment]
* pad_constant_like
cn_doc:
image
image
image

en_doc:
image
image
image

**Comment by FrostML** [comment]
* pad2d
cn_doc:
image
image

en_doc:
image
image

**Review by Boyan-Liu — DISMISSED** [review]

**Review by guoshengCS — DISMISSED** [review]

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -8722,47 +8724,37 @@ def pad2d(input,
...
     Code Examples:
         .. code-block:: python
 
-          import paddle.fluid as fluid
-          data = fluid.data(name='data', shape=[None, 3, 32, 32],
-                                   dtype='float32')
-          result = fluid.layers.pad2d(input=data, paddings=[1, 2, 3, 4],
-                                      mode='reflect')
+            import paddle.fluid as fluid
+            data = fluid.data(name='data', shape=[1, 1, 2, 3], dtype='float32')
```
Comment (guoshengCS) [inline python/paddle/fluid/layers/nn.py]: Maybe we can refine the shape to be more meaningfulr not only consistent with cn doc.

Comment (FrostML) [reply python/paddle/fluid/layers/nn.py]: Done. Thanks.

**Review by guoshengCS — APPROVED** [review]

**Review by Boyan-Liu — APPROVED** [review]

**Pull request merged**